src/nvapi/nvapi_d3d_low_latency_device.h

```diff
@@ -40,12 +40,14 @@ namespace dxvk {
       private:
         inline static std::unordered_map<IUnknown*, ID3DLowLatencyDevice*> m_lowLatencyDeviceMap;
         inline static std::unordered_map<ID3DLowLatencyDevice*, std::unique_ptr<LowLatencyFrameIdGenerator>> m_frameIdGeneratorMap;
+        inline static std::map<std::pair<IUnknown*, uint64_t>, uint64_t> m_lastFrameIdMap;
```

Comment:
Probably debatable, but I think instead of a map with pairs as keys, it would be a bit simpler if this was a map with 13-element arrays as values.